bowtie/schemas/cli/summary.json

```diff
@@ -77,32 +77,37 @@
         "unevaluatedItems": false,
         "prefixItems": [
           { "$ref": "tag:bowtie.report,2024:models:implementation:id" },
-          {
-            "description": "Unsuccessful tests for a specific implementation.",
+          { "$ref": "#/$defs/unsuccessfulTests" }
```

Comment:
Please send a PR doing these schema changes separately/immediately, as we can merge those immediately.

Reply:
Done!